### PR TITLE
Fix Backspace not working in create new key in CLI.

### PR DESCRIPTION
**Pull request by Hazealign**
This PR resolves Issue #983.
Tested in WSL2 Ubuntu 18.04, Windows 10 CMD and Powershell, macOS 10.15.
Feel free to review this. Thanks. :)

### CODE REVIEW

File: Libplanet.Tools/ConsolePasswordReader.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Text;
+
+namespace Libplanet.Tools
+{
+    public class ConsolePasswordReader
```
Comment (dahlia) [inline Libplanet.Tools/ConsolePasswordReader.cs:6]: Shouldn't this new class be used elsewhere (to replace `GetPass`)?

**Review by dahlia — COMMENTED** [review]
- As now we're no more dependent on the `GetPass` library, we should remove it from the dependencies in the *Libplanet.Tools.csproj* file.
- Could you add `[changelog skip]` into the commit message? (It's okay whether that string goes the end of the commit message or the first line.  See also this commit's message: b8048bd42480aa20b66f882c404dbf0d534430b7.)

File: Libplanet.Tools/ConsolePasswordReader.cs (hunk 1 above)
Comment (Hazealign) [reply Libplanet.Tools/ConsolePasswordReader.cs:6]: In currently, GetPass is only used on Libplanet.Tools. I think it'll be fine to be here.

File: Libplanet.Tools/ConsolePasswordReader.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                }
+                else if (!char.IsControl(key))
+                {
+                    password.Append(key);
+                    Console.Write('*');
+                }
+            }
+            while (true);
+
+            Console.WriteLine(password.ToString());
```
Comment (longfin) [inline Libplanet.Tools/ConsolePasswordReader.cs:42]: It seems expose `password` via `Console`. is it intended?

**Review by longfin — COMMENTED** [review]
Thanks for your contribution! I've left a comment.

File: Libplanet.Tools/ConsolePasswordReader.cs (hunk 2 above)
Comment (dahlia) [reply Libplanet.Tools/ConsolePasswordReader.cs:42]: Apparently it's unintended…

**Comment by dahlia** [comment]
Yesterday I asked you to add `[changelog skip]` to the commit message, but it was my mistake; this should contain a corresponding changelog in the *CHANGES.md* file.  Could you write a changelog as well? The unreleased next version at present is 0.10.0, and as it's a change about CLI tools, the changelog should go to the <q>CLI tools</q> section under the <q>Version 0.10.0</q>:

https://github.com/planetarium/libplanet/blob/938032319eaaa5caebbd1bcf78026b25ec2f7075/CHANGES.md#L244-L246

Plus, could you rebase the commits on the current main branch? I.e., `git pull --rebase git://github.com/planetarium/libplanet.git main`.

File: Libplanet.Tools/ConsolePasswordReader.cs (hunk 2 above)
Comment (Hazealign) [reply Libplanet.Tools/ConsolePasswordReader.cs]: Sorry for unintended debug code. I removed it.

**Review by dahlia — DISMISSED** [review]

**Review by dahlia — APPROVED** [review]

**Review by moreal — APPROVED** [review]

**Review by longfin — APPROVED** [review]

**Review by earlbread — APPROVED** [review]

**Review by limebell — APPROVED** [review]

**Comment by longfin** [comment]
Thanks for your contribution again!

**Pull request merged**